ui-tests/src/test/java/org/openmrs/reference/AddEditAppointmentBlockTest.java

```diff
@@ -17,61 +17,65 @@
 import static org.junit.Assert.assertTrue;
 
-
 public class AddEditAppointmentBlockTest extends ManageProviderSchedulesTest {
 	
-	
 	private final String startTimeFirstAppointment = "09";
+	
 	private String firstAppointment = "Gynecology";
-    private String secondAppointment = "Dermatology";
-    int firstAppointmentIndex = 0;
-    int secondAppointmentIndex = 1; 
+	
+	private String secondAppointment = "Dermatology";
+	
+	int firstAppointmentIndex = 0;
+	
+	int secondAppointmentIndex = 1;
 	
 	@Before
-    public void setUp() throws Exception {
-    	super.setUp();
+	public void setUp() throws Exception {
+		super.setUp();
 	}
-
-    @Test
-    @Category(BuildTests.class)
-    public void editAppointmentBlockTest() throws Exception {
-        
-        /*
-         * The logic behind the text is:
-         * create an appointment and assert that exists 
-         * change the service and assert that it has been changed
-         * search the newly created service
-         * delete the appointment
-    	*/
-    	appointmentBlocksPage.selectLocation(locationName);
-        appointmentBlocksPage.clickOnDay();
-        appointmentBlocksPage.enterStartTime(startTimeFirstAppointment);
-        appointmentBlocksPage.enterService("gyne");
-        appointmentBlocksPage.enterProvider(provider);
-        appointmentBlocksPage.clickOnSave();
-        assertTrue(appointmentBlocksPage.getServiceOfDay(firstAppointmentIndex).equals(firstAppointment));
-        appointmentBlocksPage.clickOnAppointment();
-        appointmentBlocksPage.clickOnEdit();
-        appointmentBlocksPage.removeAppointment();
-        appointmentBlocksPage.enterService("derma");
-        appointmentBlocksPage.clickOnSave();
-        assertTrue(appointmentBlocksPage.getServiceOfDay(firstAppointmentIndex).equals(secondAppointment));
-        appointmentBlocksPage.goToPage(appointmentBlocksPage.getPageUrl());
-        appointmentBlocksPage.selectLocation(locationName);
-        appointmentBlocksPage.clickOnDay();
-        //let's check that also the Cancel button works 
-        appointmentBlocksPage.clickOnCancel();
-        appointmentBlocksPage.clickOnAppointment();
-        appointmentBlocksPage.clickOnDelete();
-        //click on "left area" is needed because the tooltip shadows the delete button
-        appointmentBlocksPage.clickOnleft();
-        appointmentBlocksPage.clickOnConfirmDelete();
-    }
-    
-    @After
-    public void tearDown() throws Exception {
-    	super.tearDown();
-    }
-
+	
+	/*
+	 *@verifies the creation and editing of an appointment  
+	 */
+	@Test
+	@Category(BuildTests.class)
+	public void editAppointmentBlockTest() throws Exception {
+		
+		/*
+		 * The logic behind the text is:
+		 * create an appointment and assert that exists 
+		 * change the service and assert that it has been changed
+		 * search the newly created service
+		 * delete the appointment
+		*/
+		appointmentBlocksPage.selectLocation(locationName);
```

Comment:
Still, as said previously, have you looked at the class with the raw visualization?
However, no problem to fix the indentation, please can you point me towards a valuable code example? 
To tell the truth, I honestly expect that the code is properly formatted after the application of "openmrs formatter" (wiki page also advises to call automatically the formatter when saving). 
Don't take me wrong, if the formatter file (I mean the .xml) can't assure the expected indentation, maybe we should also take a look at it. 
Thanks